grakn-test/src/test/java/ai/grakn/test/graql/reasoner/ReasoningTests.java

```diff
@@ -46,7 +48,7 @@ public void testSet10() {
         QueryBuilder iqb = testSet10.graph().graql().infer(true);
```

Comment:
It is one of the tests Szymon wrote. More are going to follow. These are just numbered at the moment.